### PR TITLE
Add pyupgrade pre-commit hook and run on all files

### PR DESCRIPTION
**Pull request by dan98765**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,11 +16,15 @@ repos:
     -   id: pretty-format-json
         args:
         - --autofix
+-   repo: https://github.com/asottile/pyupgrade
+    sha: v1.1.1
```
Comment (jkimbo) [inline .pre-commit-config.yaml]: According to the github `v1.2.0` is the latest version. Any reason you're not using that?

Comment (dan98765) [reply .pre-commit-config.yaml]: Nope; I'll go ahead and `pre-commit autoupdate` to bump to latest.

**Comment by dan98765** [comment]
@jkimbo I bumped the version up using `pre-commit autoupdate`. Does it look OK to merge?

**Comment by jkimbo** [comment]
Thanks!

**Pull request merged**